sycl/test-e2e/InvokeSimd/Spec/ImplicitSubgroup/tuple.cpp

```diff
@@ -1,5 +1,5 @@
 // TODO: enable when Jira ticket resolved
-// XFAIL: gpu
+// XFAIL: *
```

Comment:
these tests fail when compiling, and since device features, like `gpu` are not available during building it is registered as a failure instead of xfail when in split build only mode when running the tests without this change.

Reply:
ah ok the tests compfail, thanks